Command: C:\Users\Administrator\.espressif\python_env\idf5.5_py3.11_env\Scripts\python.exe C:\Users\Administrator\esp\v5.5.1\esp-idf\tools/idf_monitor.py -p COM15 -b 115200 --toolchain-prefix riscv32-esp-elf- --target esp32c5 --revision 100 --decode-panic backtrace C:\Users\Administrator\Desktop\project\esp32good\build\esp32good.elf C:\Users\Administrator\Desktop\project\esp32good\build\bootloader\bootloader.elf --force-color -m 'C:\Users\Administrator\.espressif\python_env\idf5.5_py3.11_env\Scripts\python.exe' 'C:\Users\Administrator\esp\v5.5.1\esp-idf\tools\idf.py'
[0;33m--- Warning: GDB cannot open serial ports accessed as COMx[0m
[0;33m--- Using \\.\COM15 instead...[0m
[0;33m--- esp-idf-monitor 1.7.0 on \\.\COM15 115200[0m
[0;33m--- Quit: Ctrl+] | Menu: Ctrl+T | Help: Ctrl+T followed by Ctrl+H[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[1;31m--- Error: ClearCommError failed (PermissionError(13, 'ËÆæÂ§á‰∏çËØÜÂà´Ê≠§ÂëΩ‰ª§„ÄÇ', None, 22))[0m
[0;33m--- Waiting for the device to reconnect[0m[0;33m.[0m[0;33m[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0m--- 0x4080b934: [0;33mesp_libc_include_assert_impl[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/newlib/src/assert.c[0m[0m:[1;31m96[0m
[0m--- 0x40801f3a: [0;33mget_acquiring_dev[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_driver_spi/src/gpspi/spi_master.c[0m[0m:[1;31m700[0m
[0m--- 0x40801afa: [0;33mcheck_trans_valid[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_driver_spi/src/gpspi/spi_master.c[0m[0m:[1;31m1094[0m
[0m--- 0x420090e6: [0;33mgpio_config[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_driver_gpio/src/gpio.c[0m[0m:[1;31m417[0m
[0m--- 0x408026ec: [0;33mspi_device_polling_start[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_driver_spi/src/gpspi/spi_master.c[0m[0m:[1;31m1404[0m
[0m--- 0x42006780: [0;33mspi_init[0m[0m at [0;32mC:/Users/Administrator/Desktop/project/esp32good/main/my_lcd.c[0m[0m:[1;31m101[0m
[0m--- 0x420089b4: [0;33mgpio_set_intr_type[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_driver_gpio/src/gpio.c[0m[0m:[1;31m165[0m
[0m--- 0x420090e6: [0;33mgpio_config[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_driver_gpio/src/gpio.c[0m[0m:[1;31m417[0m
[0m--- 0x42006842: [0;33mspi_init[0m[0m at [0;32mC:/Users/Administrator/Desktop/project/esp32good/main/my_lcd.c[0m[0m:[1;31m133[0m
[0m--- 0x4200b1f2: [0;33mtask_wdt_isr[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/task_wdt/task_wdt.c[0m[0m:[1;31m476[0m
[0m--- 0x4200668e: [0;33mspi_transfer_hook[0m[0m at [0;32mC:/Users/Administrator/Desktop/project/esp32good/main/my_lcd.c[0m[0m:[1;31m37[0m
[0m--- 0x42006688: [0;33mapp_main[0m[0m at [0;32mC:/Users/Administrator/Desktop/project/esp32good/main/main.c[0m[0m:[1;31m8[0m
[0m--- 0x42016e9e: [0;33mmain_task[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/freertos/app_startup.c[0m[0m:[1;31m209[0m
[0m--- 0x408073f8: [0;33mvPortTaskWrapper[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/freertos/FreeRTOS-Kernel/portable/riscv/port.c[0m[0m:[1;31m258[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpxdwwdjv1"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpxdwwdjv1\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpog_2o6y0"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpog_2o6y0\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmppcs21gpz"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmppcs21gpz\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpys7fqup5"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpys7fqup5\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpkzd34f5g"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpkzd34f5g\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpf7v_00r4"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpf7v_00r4\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmp35b8vwt3"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmp35b8vwt3\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmp22yvjbmq"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmp22yvjbmq\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpi2zea6zm"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpi2zea6zm\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[1;31m--- Error: ClearCommError failed (PermissionError(13, 'ËÆæÂ§á‰∏çËØÜÂà´Ê≠§ÂëΩ‰ª§„ÄÇ', None, 22))[0m
[0;33m--- Waiting for the device to reconnect[0m[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m.[0m[0;33m[0m
[1;31m--- Error: ClearCommError failed (PermissionError(13, 'ËÆæÂ§á‰∏çËØÜÂà´Ê≠§ÂëΩ‰ª§„ÄÇ', None, 22))[0m
[0;33m--- Waiting for the device to reconnect[0m[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpst92bpj_"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpst92bpj_\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m.[0m[0;33m.[0m[0;33m.[0m[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpfk74b645"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpfk74b645\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0;33m.[0m[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m.[0m[0;33m.[0m[0;33m.[0m[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmp8_pjj5g2"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmp8_pjj5g2\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m.[0m[0;33m.[0m[0;33m.[0m[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmp0pssxjha"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmp0pssxjha\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0;33m.[0m[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m.[0m[0;33m.[0m[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmp7hv_dauf"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmp7hv_dauf\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0;33m.[0m[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m.[0m[0;33m.[0m[0;33m.[0m[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmph41h0e1w"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmph41h0e1w\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0;33m.[0m[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m.[0m[0;33m.[0m[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmp7pg6sfhm"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmp7pg6sfhm\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0;33m.[0m[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m.[0m[0;33m.[0m[0;33m.[0m[0;33m.[0m[0;33m.[0m[0;33m.[0m[0;33m.[0m[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmp6tquflxo"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmp6tquflxo\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0;33m.[0m[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m.[0m[0;33m.[0m[0;33m.[0m[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpxdah9p47"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpxdah9p47\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0;33m.[0m[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m.[0m[0;33m.[0m[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpfx546sb4"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpfx546sb4\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m.[0m[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m.[0m[0;33m.[0m[0;33m.[0m[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpvysxn8ev"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpvysxn8ev\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m.[0m[0;33m.[0m[0;33m.[0m[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpwu58a7j2"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpwu58a7j2\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0;33m.[0m[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m.[0m[0;33m.[0m[0;33m.[0m[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpo4pkwlyo"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpo4pkwlyo\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m.[0m[0;33m.[0m[0;33m.[0m[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpkquy3ob3"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpkquy3ob3\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0;33m.[0m[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m.[0m[0;33m.[0m[0;33m.[0m[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpgtjl_mmj"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpgtjl_mmj\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m.[0m[0;33m.[0m[0;33m.[0m[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmptku7blen"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmptku7blen\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m--- Stack dump detected[0m
[0;33m.[0m[0m--- 0x4080633a: [0;33mpanic_abort[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/panic.c[0m[0m:[1;31m483[0m
[0m--- 0x408062f8: [0;33mesp_vApplicationTickHook[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/freertos_hooks.c[0m[0m:[1;31m31[0m
[0m--- 0x40800003: [0;33m_vector_table[0m[0m at [0;32m??[0m[0m:[1;31m?[0m
[0;33m.[0m[0;33m.[0m[0;33m.[0m[0;33m--- Warning: Failed to run gdb_panic_server.py script: Command '['riscv32-esp-elf-gdb', '--batch', '-n', 'C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\esp32good.elf', '-ex', 'add-symbol-file C:\\Users\\Administrator\\Desktop\\project\\esp32good\\build\\bootloader\\bootloader.elf', '-ex', 'target remote | "C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe" -m esp_idf_panic_decoder --target esp32c5 "C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpu_eek46n"', '-ex', 'bt']' returned non-zero exit status 1.
--- Warning: b'add symbol table from file "C:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf"\r\nC:UsersAdministratorDesktopprojectesp32goodbuildbootloaderbootloader.elf: No such file or directory.\r\nerror starting child process \'C:\\Users\\Administrator\\.espressif\\python_env\\idf5.5_py3.11_env\\Scripts\\python.exe -m esp_idf_panic_decoder --target esp32c5 C:\\Users\\ADMINI~1\\AppData\\Local\\Temp\\tmpu_eek46n\': CreateProcess: No such file or directory\r\nNo stack.\r\n'

[0m
[0m--- 0x40806430: [0;33mesp_restart_noos[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_system/port/soc/esp32c5/system_internal.c[0m[0m:[1;31m144[0m
[0m--- 0x4084bba0: [0;33mesp_bootloader_get_description[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/esp_bootloader_format/esp_bootloader_desc.c[0m[0m:[1;31m39[0m
[0m--- 0x4084e5a0: [0;33mesp_flash_encryption_enabled[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader_support/src/flash_encrypt.c[0m[0m:[1;31m89[0m
[0m--- 0x4084bbaa: [0;33mcall_start_cpu0[0m[0m at [0;32mC:/Users/Administrator/esp/v5.5.1/esp-idf/components/bootloader/subproject/main/bootloader_start.c[0m[0m:[1;31m25[0m
[0;33m.[0m[0;33m[0m
[0m
